content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -16,11 +16,14 @@ Additionally, there are two special kind of files created by certain
   that form the pipeline(s) of a project, and their connections (_dependency
   graph_ or DAG).
 
+  These come with a matching [`dvc.lock` file](#dvclock-file) to track the
+  latest file versions.
```

Comment:
it tracks not only file versions

Reply:
Good point. Not even about versions either. Using "record the pipeline state" now.